backend/TreeOfAKind.Application/Query/Trees/TreeQueryValidator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentValidation;
+using TreeOfAKind.Application.Command.Trees;
+using TreeOfAKind.Application.Configuration;
+using TreeOfAKind.Domain.Trees;
+
+namespace TreeOfAKind.Application.Query.Trees
+{
+    public class TreeQueryValidator<TResponse> : AbstractValidator<TreeQueryBase<TResponse>>
```

Comment:
No need for query validation, just return `null`.

Reply:
k

Reply:
Removed validation if tree exists but ill leave validations whether id is proper id so it will be easier to spot a bug in request (for example in Datadog it will show up as an error)